includes/class-wc-reports-interval.php

```diff
@@ -0,0 +1,227 @@
+<?php
+/**
+ * Class for time interval handling for reports
+ *
+ * @package  WooCommerce/Classes
+ * @version  3.5.0
+ * @since    3.5.0
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+define( 'SECONDS_PER_DAY', 86400 );
+define( 'SECONDS_PER_HOUR', 3600 );
```

Comment:
WordPress already declare constants for it: https://github.com/WordPress/WordPress/blob/1ae9491548e64e66c3283c97dd082a691fa57e08/wp-includes/default-constants.php#L116-L134

Reply:
fixed